mc-conflicts-ui/src/conflicts/models/rootStore.ts

```diff
@@ -1,10 +1,11 @@
-import { types, Instance, getEnv } from "mobx-state-tree";
+import { types, Instance, getEnv, onAction } from "mobx-state-tree";
 import { useContext, createContext } from 'react';
 import { ConflictStore } from './conflictStore';
+import { ResponseState } from "../../common/models/responseState";
```

Comment:
`import { ResponseState } from "../../common/models/ResponseState";`

Reply:
fixed in 51f8b14